src/docs/sdk/event-payloads/lockreason.mdx

```diff
@@ -6,6 +6,12 @@ Represents an instance of a held lock (monitor object) in a thread. Typically, t
 to determine which other thread is holding the lock in case the current thread is blocked. In languages like
 `Java` or `Kotlin` this typically will be represented by a `java.lang.Object` type.
 
+<Note>
```

Comment:
I think we could even embed the interface into this page, before the `Examples` section, wdyt?

Reply:
I think this would make the page more complex than necessary, also from what I see we usually don't do that for embedded interfaces. But I'll add a link to the Threads interface for easier navigation

Reply:
We kinda do embed them, e.g. [stackframes](https://develop.sentry.dev/sdk/event-payloads/stacktrace/) or [contexts](https://develop.sentry.dev/sdk/event-payloads/contexts/), but this is also fine

Reply:
You're right, we do on some pages - IMO in those the embedded interfaces are more essential/required and the parent interface is mostly empty/a collection, so it's a bit different. I think the link is fine